todocore/src/main/java/com/knighttodo/todocore/rest/DayResource.java

```diff
@@ -147,4 +152,24 @@ public void deleteDay(@PathVariable UUID dayId) {
             throw new DayCanNotBeDeletedException("Day can't be deleted.", ex);
         }
     }
+
+    @GetMapping("/date")
+    @ResponseStatus(HttpStatus.OK)
+    @ApiOperation(value = "Find a day by Date", response = DayResponseDto.class)
+    @ApiParam(value = "Find a day by date", example = "2023-02-06", required = true, readOnly = true)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Day found"),
+            @ApiResponse(code = 400, message = "Invalid operation"),
+            @ApiResponse(code = 403, message = "Operation forbidden"),
+            @ApiResponse(code = 500, message = "Unexpected error")
+    })
+
+    public DayResponseDto findDayByDate(@RequestParam(name = "date")  @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate date) {
```

Comment:
is @RequestParam required is true by default?

Reply:
Yes, because this endpoint doesn't work without @RequestParam.

Reply:
no-no, the question was if we use @RequestParam and @RequestParam(required=true) are the same?

Reply:
I understood. @RequestParam(required=true) by default. In this case, we receive an exception when Param missing. In another case, we receive a null value. But in this case, we must handle this.